README.md

```diff
@@ -1417,6 +1417,38 @@ Other Style Guides
       return name;
     }
     ```
+  <a name=variables--no-chain-assignment""></a><a name="13.5"></a>
+  - [13.5](#variables--no-chain-assignment) Don't chain variable assignments. eslint: [`one-var`](http://eslint.org/docs/rules/one-var.html)
```

Comment:
I think it will make people think that this eslint rule enforces this part of the style guide, which it does not. More accurate would be `no-undef`, but that still does not fully enforce this. I think we should remove this reference and open an issue to either add a new rule or expand an existing rule to enforce this.